src/python/grpcio_tests/tests_aio/unit/server_test.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 The gRPC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+import logging
+import unittest
+
+from grpc.experimental import aio
+from src.proto.grpc.testing import messages_pb2
+from src.proto.grpc.testing import benchmark_service_pb2_grpc
+
+
+class BenchmarkServer(benchmark_service_pb2_grpc.BenchmarkServiceServicer):
```

Comment:
For the client, we ended up re-using protos that there were there already used for the unit test of the synchronous client. TBH I do not like very much reusing a benchmark proto and nor reusing protos that are already there and have been designed for other tests.

Having the feeling that it reduces the freedom for implementing the tests.

WDYT on starting using our own protos for the `Aio` module that are not coupled to the Benchmarking and also neither to other Tests?

Reply:
Good catch.

Stripped the ProtoBuf part out of this test. It is not necessary, and gRPC should be agnostic to underlying protocol.